lib/computer_player.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "player"
+
+class ComputerPlayer < Player
+
+  def get_input
+    random_hand_gesture
+  end
+
+  def random_hand_gesture
+    @player_input = ["rock", "paper", "scissors"].sample
```

Comment:
These variables are made into instance variables, but I can't see where else they are used in the class. Does the `["rock", "paper", "scissors"].sample` need to be set to a variable?

Reply:
isn't the variable @player_input?

Reply:
ah, @player_input isn't needed is what I think Daisy means.. could just return it or even squash this method and 'get_input' into just the one method.